examples/counter-app/src/index.py

```diff
@@ -1,30 +1,31 @@
-from pyfyre import render
 from pyfyre.nodes import *
-from pyfyre import State
+from browser import DOMEvent
+from pyfyre import render, State
+
 
 class App(Widget):
-	def __init__(self):
+	def __init__(self) -> None:
 		self.count = State[int](0)
-
 		super().__init__()
 
 	def build(self) -> Element:
-
-		def increment(e):
+		def increment(event: DOMEvent) -> None:
 			self.count.set_value(self.count.value + 1)
 
-		def decrement(e):
+		def decrement(event: DOMEvent) -> None:
 			self.count.set_value(self.count.value - 1)
 
 		return Element("main", lambda: [
 			Button(
 				onclick=decrement,
-				children=lambda: [ Text("-") ]
+				children=lambda: [Text("-")]
 			),
+			Text(" "),
 			Text(self.count),
+			Text(" "),
```

Comment:
This is weird, you can just add styles to the `Text(self.count)` element.

Reply:
How do you do that? Can you give me an example?

Reply:
I think you can do it with this:

```py
s = Style(padding_left="5px", padding_right="5px")

Text(self.count, styles=[s])
```

Reply:
`Text` is not an `Element` so we can't do that.`Text` is just a literal text in the HTML document without a tag. But we can still wrap it in an element to give it styling.

```py
s = Style(padding_left="5px", padding_right="5px")
Element("span", lambda: Text(self.count), styles=[s])
```

But it's a lot more code so I am not gonna do that.

Reply:
LGTM. We can leave it for now, I'll just change it later.